web/annotation_layer_builder.js

```diff
@@ -124,6 +127,7 @@ class DefaultAnnotationLayerFactory {
   createAnnotationLayerBuilder(
     pageDiv,
     pdfPage,
+    annotationStorage,
```

Comment:
Please note that this requires changes here as well: https://github.com/mozilla/pdf.js/blob/72d71ba6a556093a1c88525e827e59590c9151f1/web/interfaces.js#L161-L181

(This change alone probably qualifies this patch for the `[api-minor]` tag, although the general functionality added probably does so as well.)